src/pybind/mgr/cephadm/serve.py

```diff
@@ -111,6 +113,35 @@ def serve(self) -> None:
             self.log.debug("serve loop wake")
         self.log.debug("serve exit")
 
+    def _check_certificates(self) -> None:
+        cert = self.mgr.get_store('grafana_crt')
+        key = self.mgr.get_store('grafana_key')
+        if not (cert and key):
+            self.mgr.remove_health_warning('GRAFANA_CERT_ERROR')
+            return
+
+        (org, cn) = (None, None)
+        try:
+            (org, cn) = get_cert_issuer_info(cert)
```

Comment:
we don't appear to be actually using these `org`, `cn` values here. Do we want to use them to check if this is our cert or a user provided one (as you're doing in `prepare_certificates`) in order to tailor the error message? If not, i think we can probably just remove these lines that grab the values.

Reply:
yeah, variables are not used here because we would like to raise the health warning regardless of who generated the certificates. I'll remove them and just check the certificate.